internal/server/statvar/golden/search_statvar/fem.json

```diff
@@ -748,22 +764,6 @@
       "name": "Proportion of vulnerable population receiving social assistance cash benefit [Female]",
       "dcid": "sdg/SI_COV_VULN.SEX--F"
     },
-    {
```

Comment:
Related to cl/647778136 

@n-h-diaz  - Please confirm these deletions were expected

Reply:
The deletions are expected in the refresh